<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgGraphProbe</class>
 <widget class="QDialog" name="DlgGraphProbe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>338</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>2800000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableChannel"/>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Marker Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Channel Id:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboChannel">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Cursor X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelValueX">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Cursor Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelValueY">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Channel Value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelChannelY">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
